### PR TITLE
Add bump-my-version as a dependency

### PR DESCRIPTION
**Pull request by stefsmeets**
Hi all, this PR sets up automatic version bumping as part of #65

This uses [`bump-my-version`](https://github.com/callowayproject/bump-my-version), which bumps the version according to semver:

```
bump-my-version bump patch
# 0.0.20 -> 0.0.21
bump-my-version bump minor
# 0.0.21 -> 0.1.0
bump-my-version bump major
# 0.1.0 -> 1.0.0
```

It bumps the version in all files in one go.

### CODE REVIEW

**Comment by stefsmeets** [comment]
Gentle reminder, @ThijsVroegh 😅

**Comment by ThijsVroegh** [comment]
@stefsmeets Thanks!! 👍 -)  It's on my list

File: pyproject.toml [hunk 1]
```diff
@@ -94,3 +95,26 @@ show_missing = true
 
 [tool.coverage.run]
 source = ['orangecontrib/storynavigation']
+
+[tool.bumpversion]
+current_version = "0.0.20"
```
[the remark this answers is not in the document]
Comment (ThijsVroegh) [reply pyproject.toml:100]: Nevermind; I understand it from the code below..

**Review by ThijsVroegh — CHANGES_REQUESTED** [review]
These changes allow to automatically update the version number in different places in the package after the releaser sets the tag and version. After a new release, version bump, dependency pinning, and the upload to pypi all happen automatically. This solves issue #65.

I checked that there are indeed 3 places where the version is mentioned (excluding the init) 

Question:
- How is the order of first bumpen and then releasen accounted for (and not, for example, the other way around)? Or has this become irrelIevant?

**Comment by ThijsVroegh** [comment]
Ok, thanks!

**Pull request merged**